Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 The SwiftFusion Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import TensorFlow
+import XCTest
+
+import PenguinStructures
+@testable import SwiftFusion
+
+/// Test helpers.
+extension ArrayStorageImplementation {
+  /// Creates an array with `count` copies of `element`.
+  static func create(repeating element: Element, count: Int) -> Self {
+    let array = Self.create(minimumCapacity: count)
```

Comment:
Terrible name; what's the role? `result` would be better.  I usually just go for `r` in a situation like this though.

Reply:
done